app/Console/Services/Importer.php

```diff
@@ -405,15 +417,40 @@ protected function importPireps()
 
     protected function importUsers()
     {
-        /*$this->comment('--- USER IMPORT ---');
+        $this->comment('--- USER IMPORT ---');
 
         $count = 0;
         foreach ($this->readRows('pilots') as $row)
         {
+            // TODO: What to do about pilot ids
 
+            $name = $row->firstname.' '.$row->lastname;
+            $airlineid = Airline::where('icao', $row->code)->first()->id;
```

Comment:
We don't need to do lookups here. When populating the airport and ranks, I filled out `$this->airports` and `$this->ranks` (see those import functions). So we can do a lookup in those arrays, vs having to do a DB call every time. 

`$airline_id = $this->airlines[$row->code)`